### ISSUE TITLE
Stripe error when trying to donate

### ISSUE THREAD

**Issue by henare** [openaustralia/oaf#242]
A potential donor just emailed us:

```
I received this error message:

    Sorry but we are not able to provide this at the moment.
    Stripe.js token was not passed! Report this message to the site administrator.
```

**Comment by henare** [comment]
I just tried a test donation in a vanilla Chrome on Linux:

![selection_001](https://cloud.githubusercontent.com/assets/48945/18238347/5a95a92c-737f-11e6-8007-8717e5691e96.png)

**Comment by henare** [comment]
There's an update to the CiviCRM payment processor extension. Let's try that first.

We're on 1.8 and 1.9.1 is available.

**Comment by henare** [comment]
I upgraded it and then tried another donation - and got another error!

![selection_001](https://cloud.githubusercontent.com/assets/48945/18238493/da38ad4a-7380-11e6-87e8-fdeebe4c9685.png)

**Comment by equivalentideas** [comment]
Hmmm. I can't reproduce this making a one off donation @henare. I'll try with a regular donation now.

**Comment by equivalentideas** [comment]
We’ve just received a donation through the form. So this doesn't seem to be effecting everyone.

**Comment by equivalentideas** [comment]
@henare and I ran this again and saw a few issues that I've extracted.

We observed that there's an issue when you're making a payment using an email in the civi system (leading to [this error](https://github.com/openaustralia/oaf/issues/242#issuecomment-244663829)).

We didn't see [this first error again](https://github.com/openaustralia/oaf/issues/242#issuecomment-244662837) or the reported error.

All these bugs errors are different cases. It's really hard for us to see how many people this is happening to. I'm going to add some google analytics events to start tracking these errors.

It seems our choices are:

* [ ] Invest more time trying to get to the bottom of these.
* [ ] stop using civicrm to create these payments and build something that isn't so buggy ...
